test/testcafe/framework/page-objects/SelectAuthenticatorPageObject.js

```diff
@@ -46,6 +53,10 @@ export default class SelectFactorPageObject extends BasePageObject {
     return elCount === 1;
   }
 
+  getFactorCTAButtonByIndex(index) {
```

Comment:
This method and the next one refer to "CTA" and "Select" separately but seem to basically have the same code except `.textContent`. Can we rename one or both to make the distinction better?

Reply:
i believe this change was already merged in from one of glens PR's so im not sure what other tests it may affect